### ISSUE TITLE
reply doesn't work on 5.22 on my system

### ISSUE THREAD

**Issue by 2shortplanks** [doy/reply#45]
```
Mark@travis:/tmp$ reply
> 1
Use of uninitialized value $package in concatenation (.) or string at /opt/markperl/lib/site_perl/5.22.0/Reply/Plugin/Defaults.pm line 55, <STDIN> line 1.
syntax error at (eval 25) line 4, at EOF
BEGIN not safe after errors--compilation aborted at (eval 25) line 9, <STDIN> line 1.

> ^D
Mark@travis:/tmp$ perl -v

This is perl 5, version 22, subversion 0 (v5.22.0) built for darwin-2level

Copyright 1987-2015, Larry Wall

Perl may be copied only under the terms of either the Artistic License or the
GNU General Public License, which may be found in the Perl 5 source kit.

Complete documentation for Perl, including FAQ lists, should be found on
this system using "man perl" or "perldoc perl".  If you have access to the
Internet, point your browser at http://www.perl.org/, the Perl Home Page.

Mark@travis:/tmp$ perl -e 'use Reply; print $Reply::VERSION'
0.37
Mark@travis:/tmp$ ~/perl5/perlbrew/perls/perl-5.18.2/bin/reply
> 1
Use of uninitialized value $package in concatenation (.) or string at /Users/Mark/perl5/perlbrew/perls/perl-5.18.2/lib/site_perl/5.18.2/Reply/Plugin/Defaults.pm line 55, <STDIN> line 1.
syntax error at (eval 28) line 4, at EOF
BEGIN not safe after errors--compilation aborted at (eval 28) line 9, <STDIN> line 1.
Mark@travis:/tmp$ ~/perl5/perlbrew/perls/perl-5.18.2/bin/perl -e 'use Reply; print $Reply::VERSION'
0.37
```

**Comment by doy** [comment]
What's in your `~/.replyrc`? That prompt doesn't look like the default.

**Comment by 2shortplanks** [comment]
It's an empty file

**Comment by 2shortplanks** [comment]
Okay, deleting the `.replyrc` solved the problem.

I assume an empty `replyrc` doesn't load the needed package plugin...

It'd be nice to detect that.  When I get some round tuits (i.e. after
Advent) I might take a look at that.

**Comment by doy** [comment]
Fixed in 0.38.
